v1/protocols/ipfs.ts

```diff
@@ -122,25 +117,40 @@ export class IPFSProtocol implements Protocol<Static<typeof IPFSProtocolFields>>
   async sync (id: string, folderPath: string, options?: SyncOptions, ctx?: Ctx): Promise<Static<typeof IPFSProtocolFields>> {
     ctx?.logger.info('[ipfs] Sync Start')
     const mfsLocation = path.posix.join(this.mfsRoot, id)
```

Comment:
should this still be balled `mfsLocation` now that we don't use mfs?

Reply:
We're still using MFS for keeping the data pinned, we just aren't uploading files to it one at a time.